tk_toolchain/cmd_line_tools/tk_run_app/config/core/core_api.yml

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) 2016 Shotgun Software Inc.
```

Comment:
Idk if we care about this but I thought I'd note it...  Also most of the files have 2019 copyrights, which is probably accurate to when they were authored, so I left them alone.

```suggestion
# Copyright (c) 2020 Shotgun Software Inc.
```